### ISSUE TITLE
ajax="true" on b:radiobutton doesn't work

### ISSUE THREAD

**Issue by ggam** [TheCoder4eu/BootsFaces-OSP#681]
`<b:radiobutton ajax="true" />` seems to do a trick with a hidden input field wich has an oninput event.

Problem is, the browser only triggers the oninput event when text is manually written and not when the value is programmatically changed.

**Comment by stephanrauh** [comment]
The bug is slightly different.  It never came to me to use the radiobutton without an `onclick` handler. So the default behavior kicks in. That's where things start to get strange: the default behavior is encoded in the hidden input field, where it doesn't belong. It should be implemented in the radio button.

Adding insult to injury, the hidden input field renders most attributes of the radiobutton. In general, this doesn't do much harm, but it's not the desired behavior. Deriving the `RadioButtonRenderer` from the `InputTextRenderer` doesn't seem to be the good idea it seemed to be at first glance.

**Comment by stephanrauh** [comment]
I've solved the bug by making "click" the default action. I've also uploaded a new developer snapshot to Maven Central. See #369 on how to get it.

I keep the issue open for the moment because I'm unhappy about the "pass through attribute" behavior of the hidden input field. In most cases, there's no harm except the size of the generated HTML code, but this changes by adding "style='display:block'". So it's sort of a time bomb.
